### PR TITLE
Updated awssecuritylake

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 1]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
+We assume that by following the above steps, you have successfully created the required IAM roles and generated a Role ARN. In the next step, you'll grant Sumo Logic access to Amazon Security Lake to use the services.
 
-When you create an Amazon Security Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+## Configure Amazon Security Lake Setup
+
+### Step 1. Grant Sumo logic access to Amazon Security Lake console
 
-**To create an Amazon Security Lake source:**
+To grant Sumo Logic access to the Amazon Security Lake source, you need to provide the Sumo Account ID and External ID.
+1. **Account ID**. Remember to copy and secure your `Sumo Logic ID`: `************`. You would require this ID in the further steps.
```
Comment (nmohantyctr) [inline docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:49]:
```suggestion
1. **Account ID**. Remember to copy and secure your `Sumo Logic ID`: `************`. You will require this ID in further steps.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 2]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
+2. **External ID**. The External ID is formed from your Sumo Logic region identifier. It is a unique identifier that is used to authenticate the connection between your Sumo Logic account and your Amazon Security Lake source. The format of your Sumo Logic account identifier is: `SumoDeployment`: `SumoAccountId` where:
+   * `SumoDeployment` is your Sumo Logic deployment that has to be entered in lowercase such as `au`, `ca`, `de`, `eu`, `fed`, `in`, `jp`, `us1`, or `us2`. To find your deployment, see [Sumo Logic API Authentication, Endpoints, and Security](/docs/api/getting-started.md).
+   * `SumoAccountId` is the Organization ID shown on your Account Overview page in the Sumo Logic UI. You can access it by navigating to **Administration** > **Account**  > **Account Overview**.
 
+After the authentication is done successfully, you'll create the subscriber.
```
Comment (nmohantyctr) [inline docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:54]:
```suggestion
After the authentication is successful, you'll create the subscriber.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 3]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
-4. Select **Amazon Security Lake**.
-5. Enter a name for the new **Source**. A **Description** is optional.
-6. In the **Source Category**, enter any string to tag the output collected from this distinct source. (Category metadata is stored in a searchable field called **_sourceCategory**).
+1. On the **Collectors page** click **Add Source** next to a Hosted Collector.
+1. Select an existing Hosted Collector or the one you have just created.
+1. Select **Amazon Security Lake**. <br/><img src={useBaseUrl('img/send-data/aws-security-lake.png')} alt="aws-security-lake.png" width="900"/>
+1. Enter a name for the new **Source**. The **Description** is optional.
+1. In the **Source Category**, enter any string to tag the output collected from this distinct source. (Category metadata is stored in a searchable field called **_sourceCategory**).
 7. In **Fields**. Click the **+Add Field** link to add custom log metadata fields.
 8. Enter the required fields that you want to associate, each field needs a name (key) and value.
```
Comment (nmohantyctr) [inline docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:98]:
```suggestion
8. Enter the required fields you want to associate, each field needs a name (key) and value.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 4]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
+
+The next step is to update your Amazon Security Lake subscriber to fill in the subscriber endpoint on the Subscribers page.
 
 ### Step 4: Update Amazon Security Lake subscriber
 
+To update the Amazon Security Lake subscriber, follow the steps below:
 1. Navigate to the Subscribers page and click **My Subscribers**.
 2. Edit the subscriber created in the [Setup Subscriber](#step-2-set-up-subscriber-in-amazon-security-lake-console) section in step 2.
-3. Add the subscription URL copied from the Amazon Security Lake source setup in [Create Amazon Security Lake Source](#step-3-create-amazon-security-lake-source-in-sumo-logic) section.
+3. Go to the **Subscription endpoint** field which you left it blank. Paste the subscription endpoint that you copied from the Amazon Security Lake source setup. See [Step 3.10](#step-3-create-amazon-security-lake-source-in-sumo-logic) section.
```
Comment (nmohantyctr) [inline docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:118]:
```suggestion
3. Go to the **Subscription endpoint** field, which you left blank. Paste the subscription endpoint you copied from the Amazon Security Lake source setup. See [Step 3.10](#step-3-create-amazon-security-lake-source-in-sumo-logic) section.
```

**Review by nmohantyctr — CHANGES_REQUESTED** [review]

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 5]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
 
-When you create an Amazon Security Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+## Configure Amazon Security Lake Setup
+
+### Step 1. Grant Sumo logic access to Amazon Security Lake console
 
-**To create an Amazon Security Lake source:**
+To grant Sumo Logic access to the Amazon Security Lake source, you need to provide the Sumo Account ID and External ID.
+1. **Account ID**. Remember to copy and secure your `Sumo Logic ID`: `************`. You would require this ID in the further steps.
+2. **External ID**. The External ID is formed from your Sumo Logic region identifier. It is a unique identifier that is used to authenticate the connection between your Sumo Logic account and your Amazon Security Lake source. The format of your Sumo Logic account identifier is: `SumoDeployment`: `SumoAccountId` where:
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:50]: @nmohantyctr Thank you for the suggestion. However, the sentence is correct here, and there is no need for further edits. External ID is formed which means it is obtained.

File: docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md [hunk 6]
```diff
@@ -20,68 +20,100 @@ Amazon Security Lake source provides a web services interface that can be used t
...
-## Configure Amazon Security Lake Setup
+## Setup Relevant IAM Roles on Amazon Security Lake
 
+Before setting up your Amazon Security Lake account, you need to create the following IAM roles in AWS:
 
-### Step 1. Grant Sumo logic access to Amazon Security Lake console
+### Role 1 (AmazonSecurityLakeMetaStoreManager)
 
-Granting Sumo logic access to the Amazon Security Lake source  requires access to **Sumo Account ID** and **External ID**. Find the details below:
+Create the `AmazonSecurityLakeMetaStoreManager` role in AWS Identity and Access Management (IAM). The role must carry this name and is necessary for Security Lake to support extract, transform, and load (ETL) jobs on raw log and event data that it receives from sources. Without creating and assuming this role, you cannot create your data lake or query data from Security Lake. One role can be used across Regions—there's no need to create a separate role for different Regions.
```
[the remark this answers is not in the document]
Comment (JainM6) [reply docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md:29]: "that" is used here as relative clause.

**Review by nmohantyctr — APPROVED** [review]

**Pull request merged**